CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v0.7.0
+
+- Added `link_fn` option in `button_ui` to support LiveView's `live_patch`/`live_redirect`
```

Comment:
Isn't it `ui_button`?

Reply:
🤦  how hard can it be to write a good changelog :D